admin/partials/settings/auto-backup.php

```diff
@@ -17,37 +17,30 @@
 
 ob_start();
 ?>
+<h1>
+<?php
+esc_html_e( 'Auto Updates', 'boldgrid-backup' );
+?>
+</h1>
+<p>
+	<?php __( 'WordPress has the ability to auto update both itself and your plugins and themes. Keeping your software up to date is very important, and this automation helps you more easily do that.', 'boldgrid-backup' ); ?>
+</p>
+<p>
+<?php __( 'Total Upkeep adds on top of this functionality by giving you the ability to have backups made before any auto update, and by making it easier to control what is updated and when.', 'boldgrid-backup' ); ?>
+</p>
 <div class="bg-box">
 	<div class="bg-box-top">
-		<?php esc_html_e( 'Automation', 'boldgrid-backup' ); ?>
-		<span class='dashicons dashicons-editor-help' data-id='automation'></span>
+		<?php esc_html_e( 'Backup Before Updating', 'boldgrid-backup' ); ?>
 	</div>
 	<div class="bg-box-bottom">
-		<p class='help' data-id='automation'>
-			<?php
-			printf(
-				// translators: 1: HTML anchor open tag, 2: HTML anchor close tag.
-				esc_html__(
-					'Auto Backup Before Update: Automatically perform a backup before WordPress updates. When this feature is enabled, a full backup will be made during the %1$spre_auto_update action%2$s.',
-					'boldgrid-backup'
-				),
-				'<a target="_blank" href="https://developer.wordpress.org/reference/hooks/pre_auto_update/">',
-				'</a>'
-			);
-			?>
+		<p>
+		<?php __( 'Sometimes updating your software can break your site. It\'s recommended to make a backup before updates, so in the event something goes wrong, you can reasily restore your site.', 'boldgrid-backup' ); ?>
```

Comment:
Whoever wrote this file originally, decided it would be best to output bufferring, and then echo everything, and then capture it with ob_get_contents() before returning that output. 

It could be done without echoing...but only by rewriting the whole file.

Reply:
I'm not talking best practices on **echo vs output buffering**, I was asking does this line actually do anything - does it ultimately output something to the screen?

IE even if we're using output buffering, it still needs to echo.

IE the differences between these two lines in this file:
```
<?php esc_html_e( 'Backup Before Updating', 'boldgrid-backup' ); ?>

<?php __( 'Sometimes updating your software...
```

Reply:
OOOOH. I see. yeah, my bad. :D